packages/universal-components/src/Button/styles/shared.js

```diff
@@ -19,3 +19,54 @@ export const wrapperColor = {
   facebook: defaultTokens.backgroundButtonFacebook,
   google: defaultTokens.backgroundButtonGoogle,
 };
+
+export const size = {
+  small: {
+    height: parseInt(defaultTokens.heightButtonSmall, 10),
+    fontSize: parseInt(defaultTokens.fontSizeButtonSmall, 10),
+    paddingStart: {
+      paddingButton: parseInt(defaultTokens.spaceSmall, 10),
+      paddingButtonWithIcons: parseInt(defaultTokens.spaceXSmall, 10),
+      paddingButtonWithRightIcon: parseInt(defaultTokens.spaceSmall, 10),
+      paddingButtonWithLeftIcon: parseInt(defaultTokens.spaceMedium, 10),
+    },
+    paddingEnd: {
+      paddingButton: parseInt(defaultTokens.spaceSmall, 10),
+      paddingButtonWithIcons: parseInt(defaultTokens.spaceXSmall, 10),
+      paddingButtonWithRightIcon: parseInt(defaultTokens.spaceSmall, 10),
+      paddingButtonWithLeftIcon: parseInt(defaultTokens.spaceSmall, 10),
+    },
+  },
+  normal: {
+    height: parseInt(defaultTokens.heightButtonNormal, 10),
+    fontSize: parseInt(defaultTokens.fontSizeButtonNormal, 10),
+    paddingStart: {
+      paddingButton: parseInt(defaultTokens.spaceMedium, 10),
```

Comment:
It looks like there are multiple repeating values. Cannot we simplify it just be setting something like `padding` and `paddingWithIcon` constants for each size and then just use it on each side according to presence of icon.
Or is there a case when for example left padding can be different with just left icon and with both. Now I see this case only for small button, but according to my other comment I'm not sure if it's intentional.

Reply:
Yes, the small button values werent right. Just updated them.
Unfortunately there are no repeating values
Would it be a good idea ask about RN values in `plz-orbit`? Wasnt it discussed before, by any chance, dont want to duplicate question.
 It took the way too much time converting padding values into RN one by one😩

Reply:
But after update repeating values pattern is similar also for small size. So I think it can be simplified and we don't need four tokens but just setting padding according to icon presence should be fine.
I've quickly tried to simplify it a bit, you can check `feat/button-size-prop-icons-padding` branch and let me know what do you think about this approach and if there can be any possible drawbacks. 

I totally understand you pain with design tokens. But adding RN values to Orbit is still ongoing discussion. Current state is that if the value is present in tokens we should use it (with that annoying repeating parsing if it's in pixels). And if there is some repeating value missing from tokens, you can add it to `@kiwicom/universal-components/DesignTokens`.

Reply:
I understand what you meant now, with this approach it looks much better

Reply:
How could we apply these changes here, or are you planning to do PR after merging this one?

Reply:
Good question. Lets just merge this one and I'll create new small PR with update and let you know about code review.